components/groups/windows/DisabledWindowItem.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <v-card
+    flat
+    tile
+  >
+    <v-window
+      v-model="onboarding"
+      show-arrows
+    >
+      <v-window-item
+        v-for="n in length"
+        :key="`card-${n}`"
+        disabled
+      >
+        <v-card
+          color="grey"
+          height="200"
+        >
+          <v-row
+            class="fill-height"
+            align="center"
+            justify="center"
+          >
+            <h1
+              style="font-size: 5rem;"
+              class="white--text"
+            >
+              Slide {{ n }}
+            </h1>
+          </v-row>
+        </v-card>
+      </v-window-item>
+    </v-window>
+
+    <v-card-actions class="justify-space-between">
```

Comment:
Please delete this part as this actions don't work with v-window-item properties

Reply:
DONE